electrode-ota-server/test/electrode-ota-server-test.js

```diff
@@ -78,5 +78,48 @@ describe('electrode-ota-server', function () {
         }
     })));
 
+    it('should respond to newly updated service "/update_check" with modified params', request({
+        url: '/update_check?deployment_key=evHVNrwuJDVdTvpSouLxKpEyowtcxyWcyVCxnsFD&app_version=1.2.3&package_hash=&is_companion=&label=&client_unique_id=76451410-77F2-423C-836C-623AA5F586B5'
+    }, body({
+        updateInfo: {
```

Comment:
should this be  update_info or the same updateInfo?

Reply:
it's `updateInfo` as seen in the following link: https://github.com/microsoft/code-push/pull/645/files#diff-a937c637a47cbd31cbb52c89bef7d197R126